### PR TITLE
Disable chrony and systemd-timesyncd and handle different systemd service name.

### PR DESCRIPTION
**Pull request by salvatore-coppola**
Disabled chrony and systemd-timesyncd during kura installation.

### CODE REVIEW

**Comment by MMaiero** [comment]
@marcellorinaldo Can you please have a review of this?

**Comment by salvatore-coppola** [comment]
@MMaiero  do not merge, i need to do another test.

**Review by marcellorinaldo — APPROVED** [review]

**Comment by code-incubus** [comment]
Hey, @salvatore-coppola I did some checks on this PR and I've placed details about it on the Jira board.

**Comment by MMaiero** [comment]
Any news on these checks?

File: kura/org.eclipse.kura.linux.clock/src/main/java/org/eclipse/kura/linux/clock/ChronyClockSyncProvider.java [hunk 1]
```diff
@@ -95,7 +101,7 @@ private void writeConfiguration() throws KuraException {
             }
 
             if (chronyConfigLocation == null) {
-                throw new KuraException(KuraErrorCode.CONFIGURATION_ATTRIBUTE_INVALID);
+                throw new KuraException(KuraErrorCode.CONFIGURATION_ATTRIBUTE_INVALID, "chrony configuration", "....");
```
Comment (MMaiero) [inline kura/org.eclipse.kura.linux.clock/src/main/java/org/eclipse/kura/linux/clock/ChronyClockSyncProvider.java:104]: What is this? `"...."`

Comment (salvatore-coppola) [reply kura/org.eclipse.kura.linux.clock/src/main/java/org/eclipse/kura/linux/clock/ChronyClockSyncProvider.java:104]: the attribute value is too long to be printed in the log but I didn't like the placeholder {1} printed instead.

**Pull request merged**